virt/virt-4-8-release-notes.adoc

```diff
@@ -94,7 +94,8 @@ Deprecated features are included in the current release and supported. However,
 [id="virt-4-8-changes"]
 == Notable technical changes
 
-//CNV-9052 OpenShift Virtualization now automatically automatically configures IPv6 IP when running on dual-stack clusters.
+//CNV-9052 OpenShift Virtualization now automatically configures IPv6 IP when running on dual-stack clusters.
+* {VirtProductName} now configures IPv6 addresses when running on clusters that have dual-stack networking enabled. You can xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-creating-a-service-from-a-virtual-machine_virt-using-the-default-pod-network-with-virt[create a service] that uses IPv4, IPv6, or both IP address families, if dual-stack networking is enabled for the underlying {product-title} cluster.
```

Comment:
clicking `create a service` takes us to `virt-using-the-default-pod-network-with-virt.adoc` as a whole
not to the anchor (in case it should ?)

btw clicking manually on that page on the `virt-creating-a...` leads to 404 page

Reply:
Hmm, the anchor seems to be working for me. The `create a service` link takes me to the correct module. Did you use this preview link: https://deploy-preview-33466--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes.html#virt-4-8-changes

Reply:
So all is good,
I used the github "view file" which i guess is not the right thing to do then